.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: node_js
 node_js:
-  - 4    # latest 4.x release
   - 6    # latest 6.x release
   - 8    # latest 8.x release
+  - 10   # latest 10.x release
   - node # latest stable Node.js release
-matrix:
-  allow_failures:
-    - node_js: node
```

Comment:
possibly thinking it's ok to keep this `allow_failures` as we don't really have the bandwidth to keep up with all the new versions, but on the other hand it's nice to get notified as "visibly" as possible when we do run into future breakage 🤔

Reply:
I was thinking we could do like the last time. Just create an issue and add the ignore if any test breaks in the future.